CODE_OF_CONDUCT.md

```diff
@@ -25,25 +25,25 @@ To make clear what is expected, we ask all members of the community to conform t
   - [3.2 What to do if someone is in physical danger](#32-what-to-do-if-someone-is-in-physical-danger)
   - [3.3 Code of Conduct enforcement](#33-code-of-conduct-enforcement)
 - [4 Enforcement manual](#4-enforcement-manual)
-  - [4.1 The RDM Code of Conduct group](#41-the-rdm-code-of-conduct-group)
+  - [4.1 The Code of Conduct group](#41-the-code-of-conduct-group)
   - [4.2 Urgent situations: acting unilaterally](#42-urgent-situations-acting-unilaterally)
   - [4.3 Less-urgent situations](#43-less-urgent-situations)
   - [4.4 Resolutions](#44-resolutions)
 - [5 Acknowledgements](#5-acknowledgements)
 
 ## 1 Introduction
 
-The Infectious Diseases Toolkit is a community-oriented and -led project under the auspices of the ELIXIR Research Infrastructure.
+The Infectious Diseases Toolkit is a community-oriented and -led project under the auspices of the BY-COVID project.
 We value the involvement of everyone in the community.
 We are committed to creating a friendly and respectful place for learning, teaching and contributing.
 All participants in our in-person events and online communications are expected to show respect and courtesy to others at all times.
 
 To make clear what is expected, everyone participating in activities associated with the Infectious Diseases Toolkit is required to conform to this Code of Conduct.
 This Code of Conduct applies to all spaces managed by the Infectious Diseases Toolkit including, but not limited to, in-person focus groups and workshops, and communications online via GitHub. For events ELIXIR operates a [Code of Conduct](https://elixir-europe.org/events/code-of-conduct).
 
-For issues around Code of Conduct please contact [rdm-coc@elixir-europe.org](mailto:rdm-coc@elixir-europe.org). 
+For issues around Code of Conduct please contact [idtk-coc@elixir-europe.org](mailto:idtk-coc@elixir-europe.org). 
 
-Reports will be reviewed by the RDM CoC group, unless there is a conflict of interest, and will be kept confidential.
```

Comment:
I am tempted at leaving it as just the "code of conduct group" as I don't see a need for extra naming. However, I can see how it might make things a bit clearer.